app/models/feed.rb

```diff
@@ -78,15 +77,20 @@ def sanitize_text
   end
 
   def set_public_feeds_url
-    if public? && url.blank? && ENV["PUBLIC_FEEDS_URL_PREFIX"].present?
-      self.url = "#{ENV["PUBLIC_FEEDS_URL_PREFIX"]}/#{podcast.path}/#{published_path}"
+    if private?
+      self.url = nil
+    elsif ENV["PUBLIC_FEEDS_URL_PREFIX"].present? && podcast.present?
+      public_feeds_url = "#{ENV["PUBLIC_FEEDS_URL_PREFIX"]}/#{path}"
+
+      # hacky, but only default publicfeeds urls on newer records
```

Comment:
Maybe this is a terrible idea.  But I've seen `f.prxu.org` urls being used as public feeds in the wild.  And I didn't want them to suddenly have an `<atom:link>` pointing somewhere else.

Reply:
what does the conditional do here if the output is the same on either branch? is it a way to check these cases separately?

Reply:
Hrmm... I guess that was a funny way to write that.  I was checking 2 cases, but the result is the same either way:

1. Is this a newer Feed (created this October or later)?  Then default to publicfeeds.net when blank.
2. Does this feed _already_ have a publicfeeds.net url?  Then keep updating it every time it's saved, _just in case_ the file_name or slug changed.

I'll refactor that a bit to clarify.